concepticondata/conceptlists/Hartmann-2013-162.tsv

```diff
@@ -0,0 +1,163 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS	typical_context	role_frame	meaning_list	label_for_url
+Hartmann-2013-162-1	1	APPEAR	1076	EMERGE (APPEAR)	A bear appeared.	S appears	Old meaning	appear
+Hartmann-2013-162-2	2	ASK (a question)	1309	ASK (INQUIRE)	She asked me a difficult question	A asks R Q	New meaning	ask-a-question
+Hartmann-2013-162-3	3	ASK (about)			The man asked the women about a place to rent.	X asks Y about Z	New meaning	ask-about
+Hartmann-2013-162-4	4	ASK FOR	1828	ASK (REQUEST)	The boy asked his parents for money.	A asks (X) for Y	Core list	ask-for
+Hartmann-2013-162-5	5	ASSASSINATE			The sniper assassinated the politician.	A assassinates P	New meaning	assassinate
+Hartmann-2013-162-6	6	BE A DRESS			This piece of cloth is actually a dress.	X is a dress	New meaning	be-a-dress
+Hartmann-2013-162-7	7	BE A HUNTER	1053	HUNTER	This man is a hunter.	S is a hunter	Core list	be-a-hunter
+Hartmann-2013-162-8	8	BE BRIGHT	679	BRIGHT	The light is bright.	S is bright	New meaning	be-bright
+Hartmann-2013-162-9	9	BE BURDENED			The man is burdened with a problem.	A is burdened with P	New meaning	be-burdened
+Hartmann-2013-162-10	10	BE COLD	1287	COLD	The water is cold.	X is cold	New meaning	be-cold
+Hartmann-2013-162-11	11	BE DRY	1398	DRY	The ground is dry.	S is dry	Core list	be-dry
+Hartmann-2013-162-12	12	BE FAR FROM THE TRUTH			His reasoning is far from the truth.	A is far from the truth	New meaning	be-far-from-the-truth
+Hartmann-2013-162-13	13	BE GLAD			The woman is glad.	A is glad.	New meaning	be-glad
```

Comment:
"to be glad" in [Bulakh-2013-870](https://concepticon.clld.org/values/Bulakh-2013-870-103) is mapped to REJOICE 2192.

Reply:
I would agree unless someone is disturbed by the introduced inconsistency of mapping BE-adj to 'adjectival' or 'verbal' concepts (e.g., in comparison to the mappings of BE COLD, BE DRY).

Reply:
I don't think we need any new BE concepts here, DRY, COLD and REJOICE seem to fit fine. And BE FAR FROM THE TRUTH seems like a fairly rare concept.

Reply:
So I'd leave those first 3 as is.